microservices-connector/cmd/router/main.go

```diff
@@ -198,6 +199,32 @@ func executeStep(
 	return callService(step, serviceURL, input, headers)
 }
 
+func mergeRequests(respReq []byte, initReqData map[string]interface{}) []byte {
```

Comment:
Just curious, why the handleEnsemblePipeline does not need this mergeRequest?

Reply:
Hi @KfreeZ, based on current implementation, EnsemblePipeline have no response case, so just let it there.